settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:console_authorize_with_ssl_client: true
+:console_authorize_with_trusted_hosts: false
```

Comment:
I'd prefer if these were shorter, as they will need to be installer options as well.

Maybe console_ssl and console_trusted_hosts instead.